### ISSUE TITLE
Decorates main_content_renderer.html instead of replacing the service

### ISSUE THREAD

**Issue by Elendev** [chapter-three/next-drupal#131]
The Renderer https://github.com/chapter-three/next-drupal/blob/691a4b8bcf747b13e908c3dc4423ff168128d430/modules/next/next.services.yml#L11-L28 replaces the base service for HTML content renderer, which is declared in `core.services.yml`: https://api.drupal.org/api/drupal/core%21core.services.yml/service/main_content_renderer.html/9.3.x

Unfortunately it might not be deterministic: if the Drupal core service is, for any reason, declared after the module's one, it will not work as expected.

It would be better to replace it by a decorator (see https://www.drupal.org/docs/drupal-apis/services-and-dependency-injection/altering-existing-services-providing-dynamic and https://symfony.com/doc/current/service_container/service_decoration.html)

**Comment by shadcn** [comment]
Makes sense. We are looking into making the HTMLRenderer more flexible in `2.0.0`.

Happy to look into it for a `1.x.x` release if this is a blocker.

**Comment by shadcn** [comment]
Fixed by https://github.com/chapter-three/next-drupal/pull/138
